packages/snaps-execution-environments/src/common/test-utils/endowments.ts

```diff
@@ -24,14 +24,14 @@ import { SILENT_LOGGER } from './logger';
  */
 export function walkAndSearch(subject: unknown, target: unknown) {
   let result = false;
-  const walker = new LavaTube(
+  walk(
```

Comment:
Looks like this API changed with the release of LavaTube. 

@weizman Could you take a quick look and verify this usage is still fine?